scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt

```diff
@@ -0,0 +1,205 @@
+2023-10-19__16-00
+
+Files to amend
+
+Files with no bad_
+sub-P05-G2_task-oddball_run-2
+sub-P15-G4_task-oddball_run-2
+sub-P46-G3_task-oddball_run-1
```

Comment:
Should that log be committed?

Reply:
unsure, I'm not sure where I should save a log of what has been checked so far and does not need to be rechecked?

Reply:
Personally, I would not bother looking through the log and would simplfy re-run the script. You are not preloading the data in memory, so the I/O operations should be fast enough.

Reply:
still quite slow! probably specific to our storage though... but I agree, it's worth redoing to be safe

Reply:
> probably specific to our storage though

I think so.. should not take more than 1 or 2 minutes I think.